stable/datacube-dev-env/templates/ingress.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "datacube-dev-env.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+{{- $servicePort := .Values.service.port -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ .Release.Name }}
+    app.kubernetes.io/name: {{ include "datacube-dev-env.name" . }}
+    helm.sh/chart: {{ include "datacube-dev-env.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+          - path: {{ $ingressPath }}
+            backend:
+              serviceName: {{ $fullName }}
+              servicePort: {{ $servicePort }}
+  {{- end }}
+  {{- if and .Values.global.domain .Values.ingress.prefixes }}
+  {{- $domain := .Values.global.domain }}
+  {{- range .Values.ingress.prefixes }}
+    - host: "{{ . }}.{{ $domain }}"
+      http:
+        paths:
+          - path: {{ $ingressPath }}
+            backend:
+              serviceName: {{ $fullName }}
+              servicePort: {{ $servicePort }}
+  {{- end }}
+  {{- end }}
+{{- end }}
```

Comment:
Ingress is not required for a dev container

Reply:
Ingress file removed